content/en/apps/examples/covid-rdt-reference-app.md

```diff
@@ -0,0 +1,490 @@
+---
+title: "COVID-19 RDT Reference Application"
+linkTitle: "COVID-19 RDT"
+weight: 
+description: >
+ A customizable CHT application with support for 3rd party RDT integration   
+relatedContent: >
+---
+
+Medic has worked with [FIND](https://www.finddx.org) to provide a reference application of rapid diagnostic tests (RDTs) for COVID-19 in the CHT.  By leveraging the expertise of third-party applications, like [Dimagi's RD-Toolkit](https://github.com/dimagi/rd-toolkit/), you can customize this application in your current or future deployment of the CHT.
+
+You can find the code for this application in Medic's main [CHT Core repository on GitHub](https://github.com/medic/cht-core/tree/master/config/covid-19).
```

Comment:
It would be helpful for us at Medic to be less possessive of the CHT ;)

```suggestion
You can find the code for this application in the [CHT Core repository on GitHub](https://github.com/medic/cht-core/tree/master/config/covid-19).
```

Reply:
\o/